.github/workflows/cron.yml

```diff
@@ -1,30 +1,37 @@
-name: Platform Test Suite
+name: Run All Tests Every Day
 
 on:
-  release:
-    types:
-      - published
-  pull_request:
-    branches:
-      - master
-      - v[0-9]+\.[0-9]+-dev
+  schedule:
+    # TODO: Run everyday
+    - cron: '49 */160 * * *'
```

Comment:
To run daily at 4:30am GMT (just before we start work most days): `'30 4 * * *'`